### PR TITLE
Privacy UI implementation

### PR DESCRIPTION
**Pull request by C-ollins**
Requires https://github.com/planetdecred/dcrlibwallet/pull/174

- added introduction card in overview page and popup message on wallets page
- redesign privacy page to show unmixed balance, cspp server details and mixer status
- add 'new' badger on privacy list items under wallet options
- add SetupPrivacy page
- update list cards design to include divider lines and higher radius

|  |  |
|-|-|   

|  |  |
|-|-|   

|  |  |
|-|-|   

|  |  |
|-|-|

### CODE REVIEW

File: app/src/main/java/com/dcrandroid/adapter/TransactionListAdapter.kt [hunk 1]
```diff
@@ -97,10 +97,16 @@ class TransactionListAdapter(val context: Context, val transactions: ArrayList<T
             holder.icon.setImageResource(iconRes)
         }
 
-        holder.itemView.setOnClickListener {
+        holder.itemView.container.setOnClickListener {
             TransactionDetailsDialog(transaction).show(context)
         }
 
+        if (position == itemCount - 1) {
```
Comment (dreacot) [inline app/src/main/java/com/dcrandroid/adapter/TransactionListAdapter.kt:104]: Some rows don't have a divider line

Screenshot from 2021-01-14 15-03-31

File: app/src/main/java/com/dcrandroid/adapter/WalletsAdapter.kt [hunk 2]
```diff
@@ -196,17 +218,40 @@ class WalletsAdapter(val fragment: Fragment, val launchIntent: (intent: Intent,
...
 
+            popupMessage?.cancel()
+
+            val dividerWidth = fragment.context!!.resources.getDimensionPixelSize(R.dimen.wallets_menu_width)
+            val hasCheckedPrivacyPage = multiWallet.readBoolConfigValueForKey(Constants.CHECKED_PRIVACY_PAGE, false)
+
             val items = arrayOf(
                     PopupItem(R.string.sign_message, R.color.darkBlueTextColor, !wallet.isWatchingOnlyWallet),
-                    PopupItem(R.string.privacy, R.color.darkBlueTextColor, !wallet.isWatchingOnlyWallet, wallet.requiresPrivacySetup()),
+//                    PopupDivider(dividerWidth),
```
Comment (dreacot) [inline app/src/main/java/com/dcrandroid/adapter/WalletsAdapter.kt:252]: remove comment if it's not useful

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -596,5 +596,39 @@
...
+    <string name="shuffle_port">Shuffle port</string>
+    <string name="shuffle_server">Shuffle server</string>
+    <string name="keep_app_opened">Keep this app opened</string>
+    <string name="mixer_help_title">How to use the mixer?</string>
+    <string name="mixer_help_desc"><![CDATA[When you turn on the mixer, your unmixed DCRs in this wallet (unmixed balance) will be gradually mixed.<br/><br/><b><font color="#091440">Important: keep this app opened while mixer is running.</font></b><br/><br/>Mixer will automatically stop when unmixed balance are fully mixed.]]></string>
+    <string name="mixer_is_running">Mixer is running…</string>
+    <string name="keep_this_app_opened">Keep this app opened</string>
+    <string name="mixer_status_info">Mixer will automatically stop when unmixed balance are fully mixed.</string>
+    <string name="check_mixer_status">Check mixer status</string>
+    <string name="mixing_elp">Mixing…</string>
```
Comment (dreacot) [inline app/src/main/res/values/strings.xml:628]: is this meant to spell `mixing_help`? or is `elp` short for something?

File: app/src/main/java/com/dcrandroid/activities/AccountMixerActivity.kt [hunk 4]
```diff
@@ -46,227 +45,87 @@ class AccountMixerActivity: BaseActivity(), CompoundButton.OnCheckedChangeListen
...
-            tv_continue.isEnabled = mixedAccountInputHelper.validatedInput != null && unmixedAccountInputHelper.validatedInput != null
-        }
-
-        mixedAccountInputHelper.textChanged = textChanged
-        unmixedAccountInputHelper.textChanged = textChanged
+        mixed_account_label.text = wallet.accountName(mixedAccountNumber)
+        unmixed_account_label.text = wallet.accountName(unmixedAccountNumber)
 
-        val usedCSPPAccounts = wallet.findCSPPAccounts()
+//      SwitchPreference(this@AccountMixerActivity, Dcrlibwallet.walletUniqueConfigKey(wallet.id, Dcrlibwallet.AccountMixerMixTxChange), mix_tx_change)
```
Comment (dreacot) [inline app/src/main/java/com/dcrandroid/activities/AccountMixerActivity.kt:59]: remove comment if it's not useful

File: app/src/main/java/com/dcrandroid/activities/AccountMixerActivity.kt [hunk 5]
```diff
@@ -315,11 +167,11 @@ class AccountMixerActivity: BaseActivity(), CompoundButton.OnCheckedChangeListen
                     SnackBar.showError(this, R.string.no_mixable_output)
                     GlobalScope.launch(Dispatchers.Main) {
                         dialog?.dismiss()
-                        changeSwitchState(true)
+                        // off switch
```
Comment (dreacot) [inline app/src/main/java/com/dcrandroid/activities/AccountMixerActivity.kt:170]: seems like a function to turn off the mixer switch was meant to go here?

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (C-ollins) [reply app/src/main/res/values/strings.xml:628]: I abbreviated `ellipsis` as `elp`.

File: app/src/main/java/com/dcrandroid/activities/AccountMixerActivity.kt (hunk 4 above)
Comment (C-ollins) [reply app/src/main/java/com/dcrandroid/activities/AccountMixerActivity.kt:59]: It's for a pending feature.

File: app/src/main/java/com/dcrandroid/adapter/TransactionListAdapter.kt (hunk 1 above)
Comment (C-ollins) [reply app/src/main/java/com/dcrandroid/adapter/TransactionListAdapter.kt]: Fixed in recent commit.

**Review by dreacot — APPROVED** [review]

**Pull request merged**